src/query/settings/src/settings_default.rs

```diff
@@ -352,6 +352,12 @@ impl DefaultSettings {
                     possible_values: None,
                     display_in_show_settings: true,
                 }),
+                ("enable_replace_into_bloom_pruning", DefaultSettingValue {
```

Comment:
maybe we should unified a setting named `enable_bloom_pruning`, which could be used in query/replace/delete/update.

Reply:
sounds reasonable to me, maybe we can apply this idea later?

for this PR, we just need a flag to toggle the bloom_pruning behavior while executing replace-into.